src/cmd/build.rs

```diff
@@ -195,56 +217,130 @@ impl BuildCommand {
             verbosity = Verbosity::Quiet;
         }
 
-        let args = ExecuteArgs {
-            manifest_path,
-            verbosity,
-            build_mode,
-            network,
-            build_artifact: self.build_artifact,
-            unstable_flags,
-            optimization_passes,
-            keep_debug_symbols: self.keep_debug_symbols,
-            skip_linting: self.skip_linting,
-            output_type,
-        };
+        let mut build_results = Vec::new();
 
-        execute(args)
+        match self.build_all {
+            true => {
+                let workspace_members = get_cargo_workspace_members(&manifest_path)?;
+                let mut args = ExecuteArgs {
```

Comment:
We can pull this out of the match statement and modify it for each workspace member

Reply:
https://github.com/paritytech/cargo-contract/pull/647/commits/f5233c0d23194f8ec390b003f4b1ecd562d5426c